### PR TITLE
Improve site accesibility

### PR DESCRIPTION
**Pull request by MxDui**
Tasks completed as described in #7899 and I also formatted the  docs/docs/css/main.css file with prettier

### CODE REVIEW

**Review by Malvoz — APPROVED** [review]
I haven't tested it, but reviewed the HTML, looks good!

**Comment by Malvoz** [comment]
@MxDui Just a tip, if you use a [keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue) before the issue number (for example "fix #7899" it will automatically close after the maintainers merge the PR.

**Comment by MxDui** [comment]
@Malvoz Thank you for the feedback!

**Comment by Falke-Design** [comment]
@MxDui thanks for your contribution.
In Leaflet we use `tabs` as seperator and not `blanks / spaces`, so your prettified css-file is different to the other css-files (and a few other things also styled different). I don't know if this is ok here, this needs to be decided by @mourner

**Review by mourner — CHANGES_REQUESTED** [review]
Yes, please revert the whitespace formatting in the CSS file.

File: docs/docs/css/main.css
Comment (Malvoz) [inline docs/docs/css/main.css:183]: There's a spelling error "taglijne" here, should be "tagline".

**Review by Malvoz — APPROVED** [review]

**Review by mourner — APPROVED** [review]

**Pull request merged**
